index.html

```diff
@@ -9479,22 +9668,35 @@
                 </tbody>
               </table>
               <div>
-                <em>Return type:</em> <code>void</code>
+                <em>Return type:</em> <code>Promise&lt;void&gt;</code>
               </div>
             </dd>
-            <dt><dfn><code>readInto</code></dfn></dt>
+            <dt><dfn><code>waitForWritable</code></dfn></dt>
             <dd>
-              <p>Reads from <code><a>RTCQuicStream</a></code> into the buffer specified
-              by the first argument and returns the number of octets read.
-              When the <code>readInto()</code> method is called,
-              the user agent <em class="rfc2119" title="MUST">MUST</em> run the following
-              steps:</p>
-              <p>1. Let <var>stream</var> be the <code><a>RTCQuicStream</a></code> object on which
-              <code>readInto()</code> is invoked.</p>
-              <p>2. If <var>stream</var>'s <a>[[\Readable]]</a> slot is <code>false</code>, <a>throw</a> an
-              <code>InvalidStateError</code>, then abort these steps.</p>
+              <p><code>waitForWritable</code> allows the write buffer to
+              empty, resolving the promise when the data queued in the
```

Comment:
I think the write buffer can always empty, regardless of whether you call waitForWritable.  All waitForWritable can do is wait, and grow the write buffer if it's not already large enough.

Reply:
Agree.